src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -878,4 +878,8 @@ Do you want to proceed?</value>
   <data name="Operator_Overloaded" xml:space="preserve">
     <value>Operator - Overloaded</value>
   </data>
+  <data name="Operation_is_not_ready_for_0_yet_see_task_center_for_more_detail" xml:space="preserve">
+    <value>Operation is not ready for "{0}" yet.
```

Comment:
Is there a way to get the name of the operation programmatically?

Reply:
@cartermp no, but we will add this per feature so we can add something per feature if we want to.

Reply:
Great, thanks.

Reply:
@cartermp but can we do that later? my thinking is since we show this when user explicitly invoke a feature. most likely they know what feature this is showing for. probably what they don't know is why. and the message is trying to say "why"

Reply:
@cartermp The above is not correct.  It would be very simple to tell the user the name of the operation that is being blocked.  I outlined how that would be done already in this PR.  I think it would make sense to just ahve the new code do that by default since it just makes sense, is simple, and would serve as a consistent pattern for all features to follow.